package.json

```diff
@@ -5,13 +5,13 @@
   "main": "src",
   "scripts": {
     "coverage": "codecov",
-    "eslint": "eslint --ignore-pattern 'src/prettier-comments/**' 'src/**'",
+    "eslint": "eslint 'src/**'",
     "generateindexes": "SCRIPT=true node scripts/generateIndexes.js",
     "lint": "npm run eslint && npm run prettier -- --list-different",
     "lint:fix": "npm run eslint -- --fix && npm run prettier -- --write",
-    "prettier": "prettier '{.,{src,tests_config}/**}/*.{js{,on},md}' 'tests/**/jsfmt.spec.js' '!src/prettier-comments/**/*'",
+    "prettier": "prettier '{.,{src,tests_config}/**}/*.{js{,on},md}' 'tests/**/jsfmt.spec.js'",
     "test": "jest",
-    "test:all": "AST_COMPARE=true npm t -- --coverage"
+    "test:all": "FULL_TEST=1 jest"
```

Comment:
can we use `true` instead of `1`? thanks

Reply:
I used `1` as it is what's used in Prettier's scripts. It can be anything though as the variable is a string and the code just checks for a `truthy` value. It would also run full tests with `FULL_TEST=false`.

Reply:
alright, I'm ok to keep 1 then thanks for the explanation